src/services/sessionExec.js

```diff
@@ -67,6 +67,40 @@ const deleteFeedback = async ({ sessionId, feedbackId, userId }) => {
   return session
 }
 
+/**
+ * Update a feedback from a session
+ * @param {*} param0
+ */
+const updateFeedback = async ({ sessionId, feedbackId, tags, userId }) => {
+  const session = await getRunningSession(sessionId)
+
+  // ensure the user is authorized to modify this session
+  if (!session.user.equals(userId)) {
+    throw new ForbiddenError('UNAUTHORIZED')
+  }
+
+  // add new tag to a feedback
+  const index = session.feedbacks.indexOf(session.feedbacks.find(feedback => feedback.id === feedbackId))
+  session.feedbacks[index].tags = tags
+
+  // sort feedbacks
+  session.feedbacks = session.feedbacks.sort((f1, f2) => {
```

Comment:
I would handle sorting only in the frontend and keep the persisted session feedbacks in the order they arrived.

I.e., you should be able to remove this from API code and add it to the React code in a very similar manner.